tests/hmm/hmumu_utils.py

```diff
@@ -2357,9 +2379,9 @@ def dnn_variables(hrelresolution, leading_muon, subleading_muon, leading_jet, su
         "QJet1_phi": subleading_jet["phi"],
         "QJet0_qgl": leading_jet["qgl"],
         "QJet1_qgl": subleading_jet["qgl"],
-        "Higgs_m": mm_sph["mass"],
+        "Higgs_m": fixm,
```

Comment:
I think @irenedutta23 found that if the dnn variable definitions have changed, in general, she needs to retrain her DNN or restore the original variables. This seems to be one such instance.